ufl/finiteelement/mixedelement.py

```diff
@@ -56,9 +56,12 @@ def __init__(self, *elements, **kwargs):
 
         # Check that all elements use the same quadrature scheme TODO:
         # We can allow the scheme not to be defined.
-        quad_scheme = elements[0].quadrature_scheme()
-        if not all(e.quadrature_scheme() == quad_scheme for e in elements):
-            error("Quadrature scheme mismatch for sub elements of mixed element.")
+        if len(elements) == 0:
```

Comment:
Can you explain why this condition is needed for point cell?

Reply:
Vertex cells have topological dimension zero, so, thanks to this snippet
https://github.com/FEniCS/ufl/blob/3492e727f4ae3e5f2bf2d6a51e0ac37903fa9978/ufl/finiteelement/mixedelement.py#L286
the list of `sub_elements` which enters the `MixedElement` constructor ends ups being an empty list